Core/Src/flash/w25q128jvsq.c

```diff
@@ -466,25 +392,12 @@ HAL_StatusTypeDef W25Q128_OSPI_Write(OSPI_HandleTypeDef* hospi, uint8_t* pData,
 }
 
 /* Read Function */
-HAL_StatusTypeDef W25Q128_OSPI_Read(OSPI_HandleTypeDef* hospi,uint8_t* pData, uint32_t ReadAddr, uint32_t Size)
+HAL_StatusTypeDef w25q128_read(uint8_t* pData, uint32_t ReadAddr, uint32_t Size)
 {
```

Comment:
You should check that the pData buffer is big enough for Size.
Otherwise, its a buffer overflow that will cause undefined behavior...